arenadata/Dockerfile.ubuntu

```diff
@@ -29,27 +29,42 @@ RUN set -eux; \
     pip2 install allure-behave==2.4.0; \
     pip2 cache purge
 
+RUN wget https://www.python.org/ftp/python/3.9.19/Python-3.9.19.tgz -O - | tar zxf - && \
```

Comment:
Why can't we just `apt install python3`? Can we use `python3-config`/`python2-config` to link with proper libraries?

Reply:
ubuntu 22 has python 3.10 as default, but in ADB code there are explicit references to 3.9. I believe that it's another task to ease that requirement and support any of python 3.X.
That's why I decided to stick to 3.9 for now.